packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -347,31 +338,6 @@ export function TransferPanel() {
     }
 
     setTransferring(true)
-    const childChainName = getNetworkName(childChain.id)
-    const isConnectedToTheWrongChain =
-      latestChain.current?.chain?.id !== latestNetworks.current.sourceChain.id
-
-    if (isConnectedToTheWrongChain) {
-      trackEvent('Switch Network and Transfer', {
-        type: isDepositMode ? 'Deposit' : 'Withdrawal',
-        tokenSymbol: 'USDC',
-        assetType: 'ERC-20',
-        accountType: isSmartContractWallet ? 'Smart Contract' : 'EOA',
-        network: childChainName,
-        amount: Number(amount),
-        version: 2
-      })
-
-      const switchTargetChainId = latestNetworks.current.sourceChain.id
-      try {
-        await switchNetworkAsync?.(switchTargetChainId)
-      } catch (error) {
-        captureSentryErrorWithExtraData({
-          error,
-          originFunction: 'transferCctp switchNetworkAsync'
-        })
-      }
-    }
```

Comment:
it won't ever happen because the switch chain check is done before calling this function